app/views/user_mailer/account_verified.html.erb

```diff
@@ -1,15 +1,67 @@
-<p class="lead">
+<%= image_tag(
+      attachments['verified.png'].url,
+      width: 110,
+      height: 80,
+      alt: '',
+      role: 'img',
+      class: 'float-center padding-bottom-4',
+    ) %>
+
+<h1><%= message.subject %></h1>
+
+<p>
+  <%= t('user_mailer.account_verified.greeting') %>
+</p>
+<p>
+  <%= t('user_mailer.account_verified.intro', date: @date) %>
+</p>
+
+<p>
+<% if @presenter.service_provider.present? %>
+  <% if @presenter.show_cta? %>
+    <%= t('user_mailer.account_verified.next_sign_in.with_sp.with_cta', sp_name: @presenter.service_provider.friendly_name) %>
+  <% else %>
+    <%= t('user_mailer.account_verified.next_sign_in.with_sp.without_cta', sp_name: @presenter.service_provider.friendly_name) %>
+  <% end %>
+<% else %>
+  <%= t('user_mailer.account_verified.next_sign_in.without_sp', app_name: APP_NAME) %>
+<% end %>
+</p>
+
+<% if @presenter.show_cta? %>
+  <table class="button expanded large radius">
+    <tbody>
+    <tr>
+      <td>
+        <table>
+          <tbody>
+          <tr>
+            <td>
+              <%= link_to t('user_mailer.account_verified.sign_in'), @presenter.sign_in_url,
+                          target: '_blank', class: 'btn-warn', rel: 'noopener' %>
+            </td>
+          </tr>
+          </tbody>
+        </table>
+      </td>
+    </tr>
+    </tbody>
+  </table>
+  <p>
+    <%= link_to(@presenter.sign_in_url, @presenter.sign_in_url, target: '_blank', rel: 'noopener') %>
+  </p>
+<% end %>
+
+<p>
   <%= t(
-        'user_mailer.account_verified.intro_html',
-        sp_name: @sp_name,
-        app_name: APP_NAME,
-        date: @date,
+        'user_mailer.account_verified.warning_contact_us_html',
```

Comment:
This section of the diff is nonsensical; these are totally separate parts of the email.